### ISSUE TITLE
cmake policy warning CMP0012 from binary release 5.0.1 in Ubuntu 20.04

### ISSUE THREAD

**Issue by rhaschke** [assimp/assimp#3166]
assimp 5.0.1 released into Ubuntu 20.04 triggers this warning:

```
CMake Warning (dev) at /usr/lib/x86_64-linux-gnu/cmake/assimp-5.0/assimpTargets.cmake:54 (if):
  if given arguments:

    "ON"

  An argument named "ON" appears in a conditional statement.  Policy CMP0012
  is not set: if() recognizes numbers and boolean constants.  Run "cmake
  --help-policy CMP0012" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/assimp-5.0/assimp-config.cmake:1 (include)
  CMakeLists.txt:26 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

```

**Comment by ReneThrane** [comment]
I get the same issue on both windows with CMake 3.17 and on Ubuntu 20.04 with CMake 3.16.

`CMake Warning (dev) at /PersistentCache/DF-u20_04/WorkspaceInstall/1/Ubuntu/Ninja_3_1_5_6/assimp-5.0.1/lib/cmake/assimp-5.0/assimpTargets.cmake:54 (if):
  if given arguments:

    "ON"

  An argument named "ON" appears in a conditional statement.  Policy CMP0012
  is not set: if() recognizes numbers and boolean constants.  Run "cmake
  --help-policy CMP0012" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.
Call Stack (most recent call first):
  /PersistentCache/DF-u20_04/WorkspaceInstall/1/Ubuntu/Ninja_3_1_5_6/assimp-5.0.1/lib/cmake/assimp-5.0/assimp-config.cmake:1 (include)
  CMakeLists.txt:349 (_find_package)
  ThirdParty/Recipe/assimp_5_0/CMakeLists.txt:75 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.
ninja: error: 'assimp::assimp-NOTFOUND', needed by 'DemoApps/GLES2/ModelLoaderBasics/GLES2.ModelLoaderBasics_d', missing and no known rule to make it
`
The project trying to include it has `cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
` set.

Searching for CMP0012 in the issue list gives multiple hits and they all seem to be related to this.

Basically this issue prevents us from using find_package to locate assimp on both windows and ubuntu builds :(

**Comment by rhaschke** [comment]
This issue is already fixed via https://github.com/assimp/assimp/pull/2698. An Ubuntu issue is filed on [launchpad](https://bugs.launchpad.net/ubuntu/+source/assimp/+bug/1869405).
